repository/const.go

```diff
@@ -20,16 +20,26 @@ const (
 	MetadataColumnRow    = "METADATA$ROW_ID"
 	MetadataColumnTime   = "METADATA$TS"
 
-	queryCreateStream        = `CREATE STREAM IF NOT EXISTS %s on table %s`
-	queryCreateTrackingTable = `CREATE TABLE IF NOT EXISTS %s LIKE %s`
-	queryAddTimestampColumn  = `ALTER TABLE %s ADD COLUMN %s TIMESTAMP`
-	queryAddStringColumn     = `ALTER TABLE %s ADD COLUMN %s STRING`
-	queryAddBooleanColumn    = `ALTER TABLE %s ADD COLUMN %s BOOLEAN`
-	queryInsertInto          = `INSERT INTO %s %s`
-	queryInsertIntoColumn    = `INSERT INTO %s (%s) %s`
-	queryIsTableExist        = `SHOW TABLES LIKE '%s'`
-	queryGetMaxValue         = `SELECT MAX(%s) FROM %s`
-	queryGetPrimaryKeys      = `SHOW PRIMARY KEYS IN TABLE %s`
+	queryCreateStream         = `CREATE STREAM IF NOT EXISTS %s on table %s`
```

Comment:
The constants are making it really hard to read the code, my suggestion will be to inline them where they are used. 
I.e. every time you read the exec/query, you have to refer back to the const file. Takes mental energy

Reply:
that's true - we can do that